frame/transaction-payment/src/lib.rs

```diff
@@ -617,7 +617,7 @@ mod tests {
 			.execute_with(||
 		{
 			let len = 10;
-			NextFeeMultiplier::put(Fixed128::saturating_from_rational(1, 2));
+			NextFeeMultiplier::put(FixedI128::saturating_from_rational(1, 2));
```

Comment:
```suggestion
			NextFeeMultiplier::put(Multiplier::saturating_from_rational(1, 2));
```

And these for the same reason.